Assets/LeapMotion/Modules/InteractionEngine/Testing/Scripts/IETestLeapProvider.cs

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+#if LEAP_TESTS
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Interaction.Tests {
+
+  public class IETestLeapProvider : LeapProvider {
```

Comment:
This could probably go in the generic testing folder instead of the IE folder specifically and be a generic stationary pose provider.